src/esp/sim/RenderInstanceHelper.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Meta Platforms, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_SIM_RENDERINSTANCEHELPER_H_
+#define ESP_SIM_RENDERINSTANCEHELPER_H_
+
+#include <string>
+#include <vector>
+
+#include "esp/core/Esp.h"
+
+namespace esp {
+namespace scene {
+class SceneNode;
+}
+namespace sim {
+class Simulator;
+}
+}  // namespace esp
+
+namespace esp {
+namespace sim {
+
+class RenderInstanceHelper {
+ public:
+  RenderInstanceHelper(Simulator& sim, bool useXYZWOrientations = true);
```

Comment:
We should consider a name that self-documents the purpose of this class from an API user's perspective. Perhaps something along the lines of `MeshRenderer`?

Reply:
I dunno, this was the best name I could come up with to capture its very narrow scope. This class itself doesn't actually do any rendering; the user still has to call `habitat_sim.render()` or whatever.